src/components/EventCalender/Weekly/WeeklyEventCalendar.tsx

```
@@ -0,0 +1,288 @@
+
+      const holidayList: JSX.Element[] = holidays
+        .filter((holiday) => holiday.date === dayjs(date).format('MM-DD'))
+        .map((holiday) => {
+          return <HolidayCard key={holiday.name} holidayName={holiday.name} />;
+        });
+
+      return (
+        <div
+          key={index}
+          className={
+            styles.day + ' ' + (allEventsList?.length > 0 && styles.day__events)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)